core-web/libs/edit-content/src/lib/fields/dot-edit-content-binary-field/dot-edit-content-binary-field.component.ts

```diff
@@ -152,15 +154,18 @@ export class DotEditContentBinaryFieldComponent
                 tap(() => this.dotBinaryFieldStore.setStatus(BinaryFieldStatus.UPLOADING)),
                 delay(500) // Loading animation
```

Comment:
do we want this delay?

Reply:
The last time I talked to @fmontes we agreed on that. Do we want to keep it? @fmontes